fw/vhost.c

```diff
@@ -21,6 +21,9 @@
 #include <linux/slab.h>
 #include <linux/sort.h>
 
+#if defined(DEBUG)
+#define PRINT_CONFIG
```

Comment:
I think we should not need extra PRINT_CONFIG. If DBG_VHOST is specified we can print it